packages/ember-toucan-core/src/components/button.gts

```diff
@@ -76,9 +77,9 @@ export default class Button extends Component<ButtonSignature> {
 
     assert(
       `Invalid variant for Button: '${variant}' (allowed values: [${VALID_VARIANTS.join(
-        ', '
+        ', ',
```

Comment:
I believe this change was due to the latest version of prettier where trailing commas are now the default.